projects/wave-core-new/src/lib/backend/backend.model.ts

```diff
@@ -148,3 +149,16 @@ type ParamTypes = string | number | Array<ParamTypes> | { [key: string]: ParamTy
 interface Params {
     [key: string]: ParamTypes;
 }
+
+export interface TimeStepDict {
+    step: number;
+    granularity: TimeStepGranularityDict;
+}
+
+export type TimeStepGranularityDict = 'Millis' |
```

Comment:
why is Millis the only unit that is abbreviated?

Reply:
I don't know, it happened here:
https://github.com/geo-engine/geoengine/pull/110